tests/test-hoist-ansible.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash -ex
+
+source /opt/venvs/ansible/bin/activate
+
+echo "Running hoist ansible syntax test"
+ansible-playbook -i /dev/null --syntax-check $(find . tests -maxdepth 1 -type f -name \*.yml -not -name .travis.yml)
```

Comment:
shellchecks SC2046 prohibits this and requires the $() to be quoted, which borks this call as it passes in the entire find result as a single parameter.  Disabled that check below.